.github/main.go

```diff
@@ -74,7 +74,7 @@ func (ci *CI) WithPipeline(
 	} else {
 		opts.DaggerVersion = daggerVersion
 	}
-	command = fmt.Sprintf("--source=.:default --docker-cfg=file:$HOME/.docker/config.json %s", command)
+	command = fmt.Sprintf("--docker-cfg=file:$HOME/.docker/config.json %s", command)
```

Comment:
Some other places can be updated too:
- `action.yml` (which is still used by some of the workflows)
- `.dagger/mage/util` (though that code's day's are very very numbered)
- `README.md` (in `.dagger` and `sdk/python`)

Reply:
on it

Reply:
done, thanks!